twitch-chat/static/js/theme.js

```diff
@@ -0,0 +1,8 @@
+$(function(){
+	$.getJSON('/config.json', function(data){
```

Comment:
So maybe just expose the theme as its own endpoint? Seems better than exposing the whole config file. Otherwise could wrap this up into some templating inside the express app.

Reply:
Im thinking its just simpler to do it this way, but is possible to make a theme endpoint. With a config editor, i think its better to have fewer endpoints. Plus,who has access to the webserver anyways? The streamer?